main.lua

```diff
@@ -1510,6 +1511,28 @@ function circfill(cx,cy,r,col)
 
 end
 
+function help()
+	print('')
+	color(12)
+	print('commands')
+	print('')
+	color(6)
+	print('load <filename>  save <filename>')
+	print('run              resume')
+	print('shutdown         reboot')
+	print('install_demos    dir')
+	print('cd <dirname>     mkdir <dirname>')
+	print('cd ..   go up a directory')
+	print('alt+enter to toggle fullscreen')
+	print('alt+f4 or command+q to fastquit')
+	print('')
+	color(12)
+	print('alt+f4 or command+q to fastquit')
```

Comment:
duplicate line, and I'd probably use `ctrl` rather than `command` as it's more generic.